catalog/ui/docs/src/main/resources/IntContents.adoc

```diff
@@ -1,38 +1,41 @@
 
-== Overview
-
-The DDF Standard Search UI application allows a user to search for records in the local Catalog (provider) and federated sources. Results of the search are returned in HTML format and are displayed on a globe, providing a visual representation of where the records were found.
+The {branding} Standard Search UI application allows a user to search for records in the local Catalog (provider) and federated sources.
+Results of the search are returned in HTML format and are displayed on a globe, providing a visual representation of where the records were found.
```

Comment:
I'd reword this to say "are displayed as points on a globe", but if you want to just take note of it for now and do it later, I'm fine with that. Just a :thought_balloon: